src/modules/redesign/i18n/de_DE/verband.json

```diff
@@ -1,17 +1,17 @@
 {
 	"nav": {
 		"links": {
-			"/alliance_logfiles": "Protokoll",
+			"/alliance_logfiles": "Toezicht",
 			"/alliance_threads": "Forum",
 			"/alliances/{id}": "Start",
-			"/schoolings": "Lehrgänge",
-			"/verband/bereitstellungsraeume": "Bereitstellungsräume",
-			"/verband/gebaeude": "Gebäude",
-			"/verband/kasse": "Verbandskasse",
-			"/verband/mitglieder/{id}": "Mitglieder",
-			"/verband/news/{id}": "News",
-			"/verband/regeln/{id}": "Regeln",
-			"/verband/werbung": "Bewerben"
+			"/schoolings": "Opleidingen",
+			"/verband/bereitstellungsraeume": "UGS",
+			"/verband/gebaeude": "Gebouwen",
+			"/verband/kasse": "Teamkas",
+			"/verband/mitglieder/{id}": "Teamleden",
+			"/verband/news/{id}": "Nieuws",
+			"/verband/regeln/{id}": "Regels",
+			"/verband/werbung": "Adverteren"
```

Comment:
Are you sure, you wanted to edit the `de_DE`-File to `nl_NL`-Translations? :raised_eyebrow:

Reply:
Sorry, thought it was copied to nl_NL